fsw/c-blinky/main.c

```diff
@@ -0,0 +1,238 @@
+typedef unsigned int U32;
```

Comment:
This is mainly a stylistic thing, but is there any reason to not name this uint32_t. That way it matches the C99 type names